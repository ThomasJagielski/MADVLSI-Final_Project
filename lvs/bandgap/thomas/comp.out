Flattening unmatched subcell p-res20k in circuit p-res8x20k (0)(8 instances)
Flattening unmatched subcell p-res20k in circuit bandgap_pnp_thomas.spice (0)(1 instance)
Flattening unmatched subcell p-res8x20k in circuit bandgap_pnp_thomas.spice (0)(1 instance)
Equate elements:  no current cell.
Equate elements:  no current cell.

Subcircuit summary:
Circuit 1: bandgap_pnp_thomas.spice        |Circuit 2: bandgap_thomas_lvs.spice        
-------------------------------------------|-------------------------------------------
sky130_fd_pr__res_xhigh_po (2)             |sky130_fd_pr__res_xhigh_po (2)             
sky130_fd_pr__pnp_05v0 (30)                |sky130_fd_pr__pnp_05v0 (30)                
Number of devices: 32                      |Number of devices: 32                      
Number of nets: 45                         |Number of nets: 45                         
---------------------------------------------------------------------------------------
Resolving automorphisms by property value.
Resolving automorphisms by pin name.
Netlists match with 4 symmetries.
Circuits match correctly.
There were property errors.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v021 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v018 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v013 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v026 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v024 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v020 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v019 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v017 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v016 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v014 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v011 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v029 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v00 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v028 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v01 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v027 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v02 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v025 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v03 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v023 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v04 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v022 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v05 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v015 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v06 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v012 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v07 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v010 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v08 property "area" has no match in circuit 2.
Circuit 1 bandgap_pnp_thomas.spice instance sky130_fd_pr__pnp_05v09 property "area" has no match in circuit 2.
Cells have no pins;  pin matching not needed.
Device classes bandgap_pnp_thomas.spice and bandgap_thomas_lvs.spice are equivalent.
Circuits match uniquely.
Property errors were found.
The following cells had property errors: bandgap_pnp_thomas.spice
