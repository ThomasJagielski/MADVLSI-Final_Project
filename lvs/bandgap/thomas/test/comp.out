Equate elements:  no current cell.

Subcircuit summary:
Circuit 1: bandgap_pnp_thomas_test.spice   |Circuit 2: bandgap_pnp_thomas_test_lvs.spi 
-------------------------------------------|-------------------------------------------
sky130_fd_pr__pnp_05v0 (30)                |sky130_fd_pr__pnp_05v0 (30)                
Number of devices: 30                      |Number of devices: 30                      
Number of nets: 44                         |Number of nets: 44                         
---------------------------------------------------------------------------------------
NET mismatches: Class fragments follow (with fanout counts):
Circuit 1: bandgap_pnp_thomas_test.spice   |Circuit 2: bandgap_pnp_thomas_test_lvs.spi 

---------------------------------------------------------------------------------------
Net: w_n4000_n3000#                        |Net: GND                                   
  sky130_fd_pr__pnp_05v0/3 = 30            |  sky130_fd_pr__pnp_05v0/1 = 30            
  sky130_fd_pr__pnp_05v0/2 = 10            |  sky130_fd_pr__pnp_05v0/2 = 10            
                                           |  sky130_fd_pr__pnp_05v0/4 = 30            
---------------------------------------------------------------------------------------

---------------------------------------------------------------------------------------
Net: w_1153_n1847#                         |Net: __UNCONNECTED_PIN__38                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_n3847_153#                          |Net: __UNCONNECTED_PIN__36                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_n3847_1153#                         |Net: __UNCONNECTED_PIN__34                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_1153_1153#                          |Net: __UNCONNECTED_PIN__32                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_n3847_n1847#                        |Net: __UNCONNECTED_PIN__30                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_1153_n2847#                         |Net: __UNCONNECTED_PIN__28                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_153_n1847#                          |Net: __UNCONNECTED_PIN__26                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_n847_1153#                          |Net: __UNCONNECTED_PIN__24                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_n3847_n2847#                        |Net: __UNCONNECTED_PIN__22                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_n847_n2847#                         |Net: __UNCONNECTED_PIN__20                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_n2847_n2847#                        |Net: __UNCONNECTED_PIN__18                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_1153_153#                           |Net: __UNCONNECTED_PIN__16                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_153_n2847#                          |Net: __UNCONNECTED_PIN__14                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_n1847_n2847#                        |Net: __UNCONNECTED_PIN__12                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_n3847_n847#                         |Net: __UNCONNECTED_PIN__10                 
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_n1847_1153#                         |Net: __UNCONNECTED_PIN__8                  
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_153_1153#                           |Net: __UNCONNECTED_PIN__6                  
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_1153_n847#                          |Net: __UNCONNECTED_PIN__4                  
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_n2847_1153#                         |Net: __UNCONNECTED_PIN__2                  
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: w_153_153#                            |Net: net1                                  
  sky130_fd_pr__pnp_05v0/2 = 1             |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
Net: a_1330_n1670#                         |Net: __UNCONNECTED_PIN__37                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_n3670_330#                          |Net: __UNCONNECTED_PIN__35                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_n3670_1330#                         |Net: __UNCONNECTED_PIN__33                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_1330_1330#                          |Net: __UNCONNECTED_PIN__31                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_n3670_n1670#                        |Net: __UNCONNECTED_PIN__29                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_1330_n2670#                         |Net: __UNCONNECTED_PIN__27                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_330_n1670#                          |Net: __UNCONNECTED_PIN__25                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_n670_1330#                          |Net: __UNCONNECTED_PIN__23                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_n3670_n2670#                        |Net: __UNCONNECTED_PIN__21                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_n670_n2670#                         |Net: __UNCONNECTED_PIN__19                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_n2670_n2670#                        |Net: __UNCONNECTED_PIN__17                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_1330_330#                           |Net: __UNCONNECTED_PIN__15                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_330_n2670#                          |Net: __UNCONNECTED_PIN__13                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_n1670_n2670#                        |Net: __UNCONNECTED_PIN__11                 
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_n3670_n670#                         |Net: __UNCONNECTED_PIN__9                  
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_n1670_1330#                         |Net: __UNCONNECTED_PIN__7                  
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_330_1330#                           |Net: __UNCONNECTED_PIN__5                  
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_1330_n670#                          |Net: __UNCONNECTED_PIN__3                  
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_n2670_1330#                         |Net: __UNCONNECTED_PIN__1                  
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
                                           |                                           
Net: a_330_330#                            |Net: __UNCONNECTED_PIN__0                  
  sky130_fd_pr__pnp_05v0/1 = 1             |  sky130_fd_pr__pnp_05v0/2 = 1             
---------------------------------------------------------------------------------------

---------------------------------------------------------------------------------------
Net: a_330_n670#                           |(no matching net)                          
  sky130_fd_pr__pnp_05v0/1 = 1             |                                           
                                           |                                           
Net: a_n670_n1670#                         |(no matching net)                          
  sky130_fd_pr__pnp_05v0/1 = 1             |                                           
---------------------------------------------------------------------------------------

---------------------------------------------------------------------------------------
(no matching net)                          |Net: Vbep                                  
                                           |  sky130_fd_pr__pnp_05v0/3 = 1             
                                           |                                           
(no matching net)                          |Net: net2                                  
                                           |  sky130_fd_pr__pnp_05v0/3 = 1             
---------------------------------------------------------------------------------------
DEVICE mismatches: Class fragments follow (with node fanout counts):
Circuit 1: bandgap_pnp_thomas_test.spice   |Circuit 2: bandgap_pnp_thomas_test_lvs.spi 

---------------------------------------------------------------------------------------
Instance: sky130_fd_pr__pnp_05v029         |Instance: sky130_fd_pr__pnp_05v0Q22        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v028         |Instance: sky130_fd_pr__pnp_05v0Q21        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v027         |Instance: sky130_fd_pr__pnp_05v0Q20        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v025         |Instance: sky130_fd_pr__pnp_05v0Q19        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v023         |Instance: sky130_fd_pr__pnp_05v0Q18        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v022         |Instance: sky130_fd_pr__pnp_05v0Q17        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v018         |Instance: sky130_fd_pr__pnp_05v0Q16        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v015         |Instance: sky130_fd_pr__pnp_05v0Q15        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v012         |Instance: sky130_fd_pr__pnp_05v0Q14        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v010         |Instance: sky130_fd_pr__pnp_05v0Q13        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v09          |Instance: sky130_fd_pr__pnp_05v0Q12        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v08          |Instance: sky130_fd_pr__pnp_05v0Q11        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v07          |Instance: sky130_fd_pr__pnp_05v0Q10        
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v06          |Instance: sky130_fd_pr__pnp_05v0Q9         
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v05          |Instance: sky130_fd_pr__pnp_05v0Q8         
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v04          |Instance: sky130_fd_pr__pnp_05v0Q7         
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v03          |Instance: sky130_fd_pr__pnp_05v0Q6         
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v02          |Instance: sky130_fd_pr__pnp_05v0Q5         
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v01          |Instance: sky130_fd_pr__pnp_05v0Q4         
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v00          |Instance: sky130_fd_pr__pnp_05v0Q3         
  1 = 1                                    |  1 = 70                                   
  2 = 1                                    |  2 = 1                                    
  3 = 40                                   |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v026         |Instance: sky130_fd_pr__pnp_05v0Q4[0]      
  1 = 8                                    |  1 = 70                                   
  2 = 40                                   |  2 = 70                                   
  3 = 40                                   |  3 = 8                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v024         |Instance: sky130_fd_pr__pnp_05v0Q4[1]      
  1 = 8                                    |  1 = 70                                   
  2 = 40                                   |  2 = 70                                   
  3 = 40                                   |  3 = 8                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v020         |Instance: sky130_fd_pr__pnp_05v0Q4[2]      
  1 = 8                                    |  1 = 70                                   
  2 = 40                                   |  2 = 70                                   
  3 = 40                                   |  3 = 8                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v019         |Instance: sky130_fd_pr__pnp_05v0Q4[3]      
  1 = 8                                    |  1 = 70                                   
  2 = 40                                   |  2 = 70                                   
  3 = 40                                   |  3 = 8                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v017         |Instance: sky130_fd_pr__pnp_05v0Q4[4]      
  1 = 8                                    |  1 = 70                                   
  2 = 40                                   |  2 = 70                                   
  3 = 40                                   |  3 = 8                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v016         |Instance: sky130_fd_pr__pnp_05v0Q4[5]      
  1 = 8                                    |  1 = 70                                   
  2 = 40                                   |  2 = 70                                   
  3 = 40                                   |  3 = 8                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v014         |Instance: sky130_fd_pr__pnp_05v0Q4[6]      
  1 = 8                                    |  1 = 70                                   
  2 = 40                                   |  2 = 70                                   
  3 = 40                                   |  3 = 8                                    
                                           |  4 = 70                                   
                                           |                                           
Instance: sky130_fd_pr__pnp_05v011         |Instance: sky130_fd_pr__pnp_05v0Q4[7]      
  1 = 8                                    |  1 = 70                                   
  2 = 40                                   |  2 = 70                                   
  3 = 40                                   |  3 = 8                                    
                                           |  4 = 70                                   
---------------------------------------------------------------------------------------

---------------------------------------------------------------------------------------
Instance: sky130_fd_pr__pnp_05v013         |(no matching instance)                     
  1 = 1                                    |                                           
  2 = 40                                   |                                           
  3 = 40                                   |                                           
                                           |                                           
Instance: sky130_fd_pr__pnp_05v021         |(no matching instance)                     
  1 = 1                                    |                                           
  2 = 40                                   |                                           
  3 = 40                                   |                                           
---------------------------------------------------------------------------------------

---------------------------------------------------------------------------------------
(no matching instance)                     |Instance: sky130_fd_pr__pnp_05v0Q2         
                                           |  1 = 70                                   
                                           |  2 = 70                                   
                                           |  3 = 1                                    
                                           |  4 = 70                                   
                                           |                                           
(no matching instance)                     |Instance: sky130_fd_pr__pnp_05v0Q1         
                                           |  1 = 70                                   
                                           |  2 = 70                                   
                                           |  3 = 1                                    
                                           |  4 = 70                                   
---------------------------------------------------------------------------------------
Netlists do not match.
The top level cell failed pin matching.
